user_interaction.py

```diff
@@ -1,110 +1,108 @@
 #!/usr/bin/env python
 import os
 
-# Determining the way the library will be used
-modeOptions = {1 : "train", 2 : "eval", 3 : "sample"}
-
-print("You want to train, evaluate or sample a model?\n\
-[1] train\n[2] evaluate\n[3] sample")
-while True:
-    mode = input()
-    if mode not in modeOptions:
-        print("Insert only 1, 2 or 3")
-        continue
-    else:
-        mode = modeOptions[mode]
-        break
-
-print("You chose to " + mode + " the model.\n")
-
-# Determining if the image will run locally or remotely
-placeOptions = {1 : "locally", 2 : "on the Google Cloud AI Platform"}
-print("You want to " + mode + " locally or on the Google Cloud AI Platform?\n\
-[1] locally\n[2] Google Cloud AI Platform")
-while True:
-    place = input()
-    if place not in [1, 2]:
-        print("Insert only 1 or 2")
-        continue
-    else:
-        break
-
-print("You chose to " + mode + " " + placeOptions[place] + "\n")
-
-# Determining path for data retrieval
-dataPath = raw_input("Please insert a path for data retrieving. This can be a local directory or a GCS Bucket:\n")
+# # Determining path for data retrieval
+data_path = raw_input("Insert a path for data retrieving. \
+This can be a local directory or a GCS Bucket:\n")
 
 # Determining path for storing model, snapshots and summaries
-storingPath = raw_input("\nPlease insert a path for storing the model, snapshots and summaries.\
- This can be a local directory or a GCS Bucket:\n")
+storing_path = raw_input("\nInsert a path for saving the model, \
+snapshots and summaries. This can be a local directory or a GCS Bucket:\n")
 
-# Determining path for recovering a snapshot
-recoveryPath = raw_input("\nPlease insert path for recovering a snapshot or a completed model.\
- This can be a local directory or a GCS Bucket:\n")
+recovery_path = raw_input("\nInsert a path from which checkpoints will\
+ be restored before training. Skip for using the same path as for saving:\n")
+if recovery_path == "":
+    recovery_path = storing_path
 
-imageURI = raw_input("\nPlease insert a IMAGE URI:")
+gin_path = raw_input("\nInsert an additional gin file search path or skip\
```

Comment:
As Jesse recommended me in my first PR, I think it is better to format the strings in this way:
('Some very '
'long string.')
instead of using \ to brake line.